src/controller/config.js

```diff
@@ -66,12 +64,24 @@ bodyTagBtn.addEventListener('click', async (event) => {
     event.stopPropagation()
     let txt = bodyTagTxt.value
     if (txt) {
-        let bodyTagId = await window.api.generateId()
+        let wasEdited = bodyTagId != null
+        if(wasEdited){
```

Comment:
if the bodyTag was edited keep the get the coordinates from the body part map. We can reuse the coordinates since it was just a name change